vscode-extension/src/aiConfigEditor.ts

```diff
@@ -193,13 +193,12 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
             return;
           }
 
-          // TODO: Should we skip events with no content changes?
-          // if (e.contentChanges.length === 0) {
-          //   console.log(
-          //     `changeDocumentSubscription ${e.document.uri} - skipping event because there are no content changes`
-          //   );
-          //   return;
-          // }
+          if (e.contentChanges.length === 0) {

```

Comment:
This is perfectly reasonable as a solution. The main thing to validate would be other kinds of document changes, like renames or file moves, to see if those trigger any weird cases. But those are much less likely than this scenario I think

Reply:
Just tested -- both rename and move fully reload the webview, so it re-calls /load and sets the config from that